tiledb/sm/query/query_condition.h

```diff
@@ -51,6 +51,65 @@ enum class QueryConditionCombinationOp : uint8_t;
 
 class QueryCondition {
  public:
+  /* ********************************* */
```

Comment:
FYI: This is a copy+paste to move the `Clause` struct from the `private` to `public` scope.